nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/RestartClusterCommand.java

```diff
@@ -88,13 +86,21 @@
 
     @Param(optional = true, defaultValue = "false")
     private boolean verbose;
-    
+
     @Param(optional = true, defaultValue = "true")
     private boolean rolling;
-    
+
     @Param(optional = true, defaultValue = "0")
     private String delay;
 
+    @Min(message = "Timeout must be at least 1 second long.", value = 1)
+    @Param(optional = true, defaultValue = "600")
```

Comment:
Not sure this should have a default.
What happens now is if run these commands it the timeout will always be 10 minutes unless you specify differently, even if you've configured the global system/env property to something else.